### PR TITLE
chore(localization): update strings in Registration page (resolves #792)

### LINKED ISSUE #792: Update text for Registration
https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=4%3A137

### PR DESCRIPTION
**Pull request by chosww**
Explain what this PR does.

Update strings in the registration work flow.

### Prerequisites

If this PR changes PHP code or dependencies:

- [x] I've run `composer format` and fixed any code formatting issues.
- [x] I've run `composer analyze` and addressed any static analysis issues.
- [x] I've run `php artisan test` and ensured that all tests pass.
- [x] I've run `composer localize` to update localization source files and committed any changes.

If this PR changes CSS or JavaScript code or dependencies:

- [x] I've run `npm run lint` and fixed any linting issues.
- [ ] I've run `npm run build` and ensured that CSS and JavaScript assets can be compiled.

### CODE REVIEW

File: resources/lang/en/passwords.php [hunk 1]
```diff
@@ -13,8 +13,8 @@
     |
     */
 
-    'reset' => 'Your password has been reset!',
-    'sent' => 'We have emailed your password reset link!',
+    'reset' => 'You have successfully reset your password for The Accessibility Exchange.',
+    'sent' => 'We have emailed you a password reset link for Accessibility Exchange. Please check your email.',
```
Comment (greatislander) [inline resources/lang/en/passwords.php]:
```suggestion
    'sent' => 'We have emailed you a password reset link for The Accessibility Exchange. Please check your email.',
```

File: resources/views/auth/register/steps/2.blade.php [hunk 2]
```diff
@@ -4,12 +4,17 @@
...
+            </p>
+        </x-expander>
         <x-hearth-radio-buttons name="context" :options="[
             [
                 'value' => 'individual',
-                'label' => __('As an individual'),
+                'label' => __('Individual'),
                 'hint' => __(
-                    'I have lived experience of being disabled or Deaf, and I want to work on accessibility projects.',
+                    'I am someone who has lived experience of being disabled or Deaf, or I am a family member or supporter or a person who is disabled or Deaf.',
```
Comment (greatislander) [inline resources/views/auth/register/steps/2.blade.php]:
```suggestion
                    'I am someone who has lived experience of being disabled or Deaf, or I am a family member or supporter of a person who is disabled or Deaf.',
```

**Review by greatislander — CHANGES_REQUESTED** [review]

File: resources/lang/en/passwords.php (hunk 1 above)
Comment (greatislander) [inline resources/lang/en/passwords.php]:
```suggestion
    'sent' => 'We have emailed you a password reset link for The Accessibility Exchange. Please check your email.',
```

**Review by greatislander — APPROVED** [review]

**Comment by cherylhjli** [comment]
Please refer to the colour coding on the latest wireframes for strings that are not matching the wires.

**Comment by greatislander** [comment]
@chosww ☝🏻

**Comment by cherylhjli** [comment]
https://staging.accessibilityexchange.ca/en/register?step=3
Email address 
Current: “This is also the email you use to log into this account. If you change this, you are also changing your log in email.”
Change to: “This is the email address you will use to sign in to The Accessibility Exchange.”

When registering as a CommOrg - selecting type of organization
- “Civil society organization”, not “broader civil society organizations”

When registering as a CommOrg - entering org name:
- the titles are: “Create new support organizations”, “create new representative organizations”, “create new broader civil society organizations” - drop the ’s’ at the end

**Pull request merged**